acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+    "train_images = sorted(glob.glob(os.path.join(data_root, \"imagesTr\", \"*.nii.gz\")))\n",
+    "train_labels = sorted(glob.glob(os.path.join(data_root, \"labelsTr\", \"*.nii.gz\")))\n",
+    "data_dicts = [{\"image\": image_name, \"label\": label_name} for image_name, label_name in zip(train_images, train_labels)]\n",
+    "train_files, val_files = data_dicts[:-9], data_dicts[-9:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Export the trt model with API\n",
+    "\n",
+    "Export the downloaded bundle model to TensorRT engine-based TorchScript models. It takes a while for converting and will generate the `model_trt_fp32.ts` and `model_trt_fp16.ts` into the `model` folder.\n",
+    "\n",
+    "The command is quite similar to the `ckpt_export` command, except the `precision` and `dynamic_batchsize` parameters. \n",
+    "\n",
+    "The `precision` parameter specifies the converted model weight precision which currently supports `fp32` and `fp16`. The `dynamic_batchsize` parameter is a sequence with three elements to define the batch size range of the input for the model to be converted. Should be a sequence like `[MIN_BATCH, OPT_BATCH, MAX_BATCH]`. After conversion, the batch size of model input should be between `MIN_BATCH` and `MAX_BATCH`. The `OPT_BATCH` is the best performance batch size that the TensorRT tries to fit. It should be the most frequently used input batch size in the application."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd spleen_ct_segmentation\n",
+    "!python -m monai.bundle trt_export network_def --filepath models/model_trt_fp32.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp32 --dynamic_batchsize \"[1, 4, 8]\"\n",
+    "!python -m monai.bundle trt_export network_def --filepath models/model_trt_fp16.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp16 --dynamic_batchsize \"[1, 4, 8]\"\n",
+    "%cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the torch model and TensorRT model\n",
+    "\n",
+    "Load the original torch model and the TensorRT engine-based TorchScript models. Please note that `torch_tensorrt>=1.4.0` must be imported before loading TensorRT engine-based TorchScript models, since it will provide some runtime context."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_parser(meta_config, inference_config):\n",
+    "    parser = monai.bundle.ConfigParser()\n",
+    "    parser.read_meta(meta_config)\n",
+    "    parser.read_config(inference_config)\n",
+    "    return parser\n",
+    "\n",
+    "\n",
+    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
```

Comment:
bundle already defines the `device` in it.
I would suggest to use `device = parser.get_parsed_content("device")` instead.

Thanks.